### DISCUSSION TITLE
Feature idea: integrate ReBAC Authorization

### DISCUSSION THREAD

**Discussion by umerkle** [abpframework/abp#18613] [Ideas]
If one wants to give authorizations based on relations between entities (e.g. based on position in an organizational hierarchy), a [ReBAC](https://en.wikipedia.org/wiki/Relationship-based_access_control) based authorization (a.k.a fine grained authorization) method/system could be of help.
There are some projects outside there like [OpenFGA](https://openfga.dev/docs/authorization-and-openfga) (or see Google Zanzibar).
I think this could bring a great feature int abp applications.

What do you think about such an integration?

**Comment by maliming** [comment]
Abp's authorization system supports customization. You can try to integrate it first.

I think frameworks should not introduce uncommon and complex features.
